WooCommerce/Classes/ViewModels/Order Details/Shipping Labels/ShippingLabelPrintingInstructionsView.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+/// Displays an illustration at the top and four steps on how to print a shipping label on an iOS device.
+struct ShippingLabelPrintingInstructionsView: View {
+    var body: some View {
+        ScrollView {
+            VStack {
+                Spacer()
+                    .frame(height: 20)
+                Image("woo-shipping-label-printing-instructions")
+                Spacer()
+                    .frame(height: 30)
```

Comment:
Given that creating views is supposed to be almost free (according to Apple), would it make sense to extract all the step views to a separate view? That way we would also remove the knowledge of the view model creation from here

Reply:
Yea it makes sense to keep the top-level view cleaner, updated in: https://github.com/woocommerce/woocommerce-ios/pull/3285/commits/b2df64348e583313ce738431b2e01ebbaf30dc2d